File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = false;
        int countP=0;
        int countY=0;
        String str = s.toLowerCase();
        String[] strArr;
        strArr = str.split("");
        for(int i=0; i<strArr.length; i++){
            if(strArr[i].equals("p"))
                countP++;
            if(strArr[i].equals("y"))
                countY++;
        }
        
        System.out.println(countP +""+ countY);
        if((countP == countY) || (countP==0 && countY==0))
            answer = true;
        return answer;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;
    let p = 0;
    let y = 0;
    s.split("").map((item) =>{
        if(item == 'p' || item == 'P' ){
            p++;
        }else if(item == 'y' || item == 'Y'){
            y++;
        }
    })
    if(p==y){
        return true;
    }else{
        return false;
    }
}